### DISCUSSION TITLE
BasicForm 能否设置为只读，disabled不能复制

### DISCUSSION THREAD

**Discussion by wsylp** [vbenjs/vue-vben-admin#1620] [Q&A]
BasicForm 能否设置为只读，disabled不能复制，看了文档没有找到相关方法

**Comment by chen0610** [comment]
设置readonly，示例
{
    field: 'city',
    component: 'Input',
    label: '所在地区',
    colProps: { span: 18 },
    componentProps:{
      readonly:true
    }
  },
